tensor/stats/stats/README.md

```diff
@@ -1,36 +1,92 @@
 # stats
```

Comment:
Does this really need to be double nested? (`stats/stats`)

Reply:
there are several packages all of the general stats variety.

Reply:
but we could discuss whether having a big list at top level might be better?  already kind of a long list -- i"m not sure it is better.